sources/packages/backend/libs/integrations/src/services/disbursement-schedule/disbursement-schedule.service.ts

```diff
@@ -199,4 +200,68 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
     const repository = disbursementScheduleRepo ?? this.repo;
     return repository.update({ id: In(disbursementIds) }, { dateSent });
   }
+
+  /**
+   * Fetch the eligible COEs for a given date.
+   * @param generatedDate Date in which the eligible COE is required for an institution.
+   * @returns eligible COEs.
+   */
+  async getEligibleCOE(
```

Comment:
"Eligible" seems to be too generic "eligible for what?".
Can we rename this for something like `getCOEOnRequestStatus` or `getPendingCOEs`?